plugin-hrm-form/src/utils/audioNotifications.ts

```diff
@@ -97,24 +103,16 @@ const notifyReservedTask = reservation => {
       );
     }
 
-    const stopAudio = () => AudioPlayerManager.stop(media);
+    const stopAudio = () => media && AudioPlayerManager.stop(media);
 
     const taskStatuses = ['accepted', 'canceled', 'rejected', 'rescinded', 'timeout'];
     taskStatuses.forEach(status => {
       reservation.on(status, stopAudio);
     });
 
-    setTimeout(stopAudio, 120000);
-
-    const checkForReservedTask = () => {
-      if (reservation.task.status === 'reserved') {
-        setTimeout(checkForReservedTask, 5000);
-      } else {
-        stopAudio();
-      }
-    };
-
-    checkForReservedTask();
+    const checkForPendingReservation = () =>
+      reservation.status === 'pending' ? setTimeout(checkForPendingReservation, 5000) : stopAudio;
```

Comment:
I think you want to call `stopAudio` here

```suggestion
      reservation.status === 'pending' ? setTimeout(checkForPendingReservation, 5000) : stopAudio();
```